evm-contracts/src/v0.6/dev/FluxAggregator.sol

```diff
@@ -670,6 +709,34 @@ contract FluxAggregator is AggregatorInterface, Owned {
     return rounds[uint32(_roundId)].updatedAt;
   }
 
+  /**
+   * @dev Internal implementation of getRound
+   */
+  function _getRound(uint256 _roundId)
+    internal
+    view
+    returns (
+      uint256 roundId,
+      int256 answer,
+      uint64 startedAt,
+      uint64 updatedAt,
+      uint256 answeredInRound
+    )
+  {
+    if (_roundId == uint256(~0)) {
+      _roundId = latestRoundId;
+    }
+
+    Round memory r = rounds[uint32(_roundId)];
+    return (
+      _roundId,
```

Comment:
Is this necessary to return the round ID that was simply passed in?

Reply:
Answering my own question: in case the user passed in INT256_MAX, we'd return the actual latest round ID.